src/main/java/uk/gov/pay/products/filters/LoggingMDCRequestFilter.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.pay.products.filters;
+
+import org.slf4j.MDC;
+
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerRequestFilter;
+import java.io.IOException;
+import java.util.Optional;
+
+import static uk.gov.pay.logging.LoggingKeys.*;
```

Comment:
make sure your intellij settings are correct to avoid wildcard imports https://gds-way.cloudapps.digital/manuals/programming-languages/java.html#imports

Reply:
Fixed